.github/workflows/galata-update.yml

```diff
@@ -0,0 +1,115 @@
+name: Update Playwright Snapshots
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-galata-snapshots:
+    name: Update Galata References
+    if: ${{ github.event.issue.pull_request && (contains(github.event.comment.body, 'update galata snapshots') || contains(github.event.comment.body, 'update snapshots')) }}
```

Comment:
Nice! Can we have a triggering sentinel word as well that makes it clear we are asking for the system to do something, kind of like mentioning meeseeksdev is a triggering keyword that doesn't come up in normal conversation? The phrase "update snapshots" seems like it might come up in normal conversation.

Reply:
Yes, that's better, thanks!